src/pages/trackfood.js

```diff
@@ -1,18 +1,33 @@
 import Head from 'next/head';
 
 import FoodForm from '../components/FoodForm';
-import Layout from '../components/Layout';
-import {StyledTitle} from '../components/styled/StyledTitle';
+import {StyledNavUl} from '../components/styled/StyledNav';
+import {StyledNavbar} from '../components/styled/StyledNew';
 
 export default function AboutPage() {
 	return (
-		<Layout>
+		<>
 			<Head>
-				<title key="title">Food Tracker</title>
-				<meta key="description" name="Food tracker" content="track your food and macro" />
+				<title key="title">Nutri Trackr</title>
+				<meta
+					key="You can save here what you eat today"
+					name="Food tracker"
+					content="track your food and macros"
+				/>
 			</Head>
-			<StyledTitle>Food Formular</StyledTitle>
+			<StyledNavUl>
+				<StyledNavbar as="a" href="/">
+					Home
+				</StyledNavbar>
```

Comment:
if you use this more than once make it a component and use it anywhere you need it 🐜

Reply:
Thanks!!